libraries/bfd_beacon.rb

```diff
@@ -41,7 +42,10 @@ def action_start
 
       converge_by("starting BFD #{new_resource.name}") do
         notifying_block do
-          bash "#{bfd_beacon}#{controls}#{listeners}"
+          code_snippet = "#{bfd_beacon}#{controls}#{listeners}"
```

Comment:
Why the `code_snippet`; this looks to be used for the `name` and `code` but one could do:
`bash <command>`. (No need for the do and repetitive `code` attribute?)